src/core/Aggregators.java

```diff
@@ -34,7 +33,7 @@
     LERP,   /* Regular linear interpolation */
     ZIM,    /* Returns 0 when a data point is missing */
     MAX,    /* Returns the <type>.MaxValue when a data point is missing */
-    MIN     /* Returns the <type>.MinValue when a data point is missing */
+    MIN,    /* Returns the <type>.MinValue when a data point is missing */
```

Comment:
Ah right yeah these are already in there at https://github.com/OpenTSDB/opentsdb/blob/next/src/core/Aggregators.java#L666-L708 in 2.3.0. I'll close this PR then.

The code for those aggregators is correct. The "First" aggregator fetches the first value from the iterator then iterates over the others to consume the iterator but doesn't assign the output anywhere. The "Last" aggregator iterates similarly but always assigns the iteration value to the variable.